tests/cli/test_cli.py

```diff
@@ -1,24 +1,26 @@
+from pathlib import Path
+import shutil
 from typing import Callable
-from _pytest.pytester import RunResult
+from _pytest.pytester import RunResult, Testdir
 import pytest
 import sys
 
 
-def test_cli_start(run: Callable[..., RunResult]):
+def test_cli_start_is_fast(testdir: Testdir):
```

Comment:
very nice! Thanks! (cc @alwx Not sure what's the latest regarding the `test_import_time` test, but in any case we can drop it now)

Does it make sense to also test for numpy or is that neglible? 

And can we add some instructions how to fix this test in case it's failing? E.g. `If this is failing this means, that a simple "rasa --help" commands imports tensorflow which makes our CLI extremely slow. In case this test is failing you've very likely added a global import of "tensorflow" which should be avoided. Consider making this import (or the import of its parent module) a local import."`

Reply:
will add the instructions, we can check for numpy in a separate issue - didn't look into it (and can't do it now)